libafl/src/stages/tmin.rs

```diff
@@ -134,6 +143,7 @@ where
                     if feedback.is_interesting(state, manager, &input, observers, &exit_kind)? {
                         // we found a reduced corpus entry! use the smaller base
                         base = input;
+                        base_post = Some(post.clone());
```

Comment:
is it possible to avoid this clone() ?

Reply:
The function `MutatedTransformPost.post_exec` consumes self. For [example](https://github.com/AFLplusplus/LibAFL/blob/53bfae0a088f4a5dcaa9e90fb3357dd8d049b0c5/libafl/src/mutators/string.rs#L53-L56), the post is a metadata, and its ownership is transferred to the metadata map in the `post_exec`. So I guess it can't be avoided?

Reply:
I think you could set a bool to `true` here and then do `base_post` conversion and `post_exec` below, after calling `post.post_exec`

Reply:
So do we still need to clone? I think the post object is no longer valid after post_exec, because it is comsumed.

Reply:
ooh...
Well I guess in that case not much we can do..

Reply:
no i removed it